Procurement/ViewModel/ItemDisplayViewModel.cs

```diff
@@ -57,7 +57,7 @@ public SolidColorBrush ItemFilterBrush
                                         : new SolidColorBrush(Colors.Transparent); }
         }
 
-        public bool IsStackSizeVisible => Item is StackableItem;
+        public bool IsStackSizeVisible => Item?.StackSize > 1;
```

Comment:
You won't see it if your stack size is one, which is a valid amount of stacks.

Reply:
Bah, nice catch I'll fix it.